sdks/python/apache_beam/ml/inference/vertex_ai_inference.py

```diff
@@ -66,26 +66,35 @@ def __init__(
       project: str,
       location: str,
       experiment: Optional[str] = None,
+      network: Optional[str] = None,
       **kwargs):
     """Implementation of the ModelHandler interface for Vertex AI.
     **NOTE:** This API and its implementation are under development and
     do not provide backward compatibility guarantees.
     Unlike other ModelHandler implementations, this does not load the model
     being used onto the worker and instead makes remote queries to a
     Vertex AI endpoint. In that way it functions more like a mid-pipeline
-    IO. At present this implementation only supports public endpoints with
-    a maximum request size of 1.5 MB.
+    IO. Public Vertex AI endpoints have a maximum request size of 1.5 MB.
+    If you wish to make larger requests and use a private endpoint, provide
+    the Compute Engine network you wish to use.
+
     Args:
       endpoint_id: the numerical ID of the Vertex AI endpoint to query
       project: the GCP project name where the endpoint is deployed
       location: the GCP location where the endpoint is deployed
       experiment (Optional): experiment label to apply to the queries
+      network (Optional): the full name of the Compute Engine network
```

Comment:
just to make sure, this is network and not subnetwork? https://cloud.google.com/vpc/docs/vpc#vpc_networks_and_subnets

Would it make sense to add an example akin to: https://github.com/apache/beam/blob/master/sdks/python/apache_beam/options/pipeline_options.py#L1045-L1046

Reply:
This is network and not subnetwork, following the aiplatform documentation (https://cloud.google.com/python/docs/reference/aiplatform/latest/google.cloud.aiplatform#google_cloud_aiplatform_init). 

Yeah the docstring getting an example makes sense, let me do that

Reply:
Done